update roll editor page with new info

JayAndromeda opened the pull request:
I wasn't too sure how you wanted the page to look so I started with a little info on navigating the roll editor. I added anchors to some of the other headers as I imagine this page can and will get quite long if you wanted say the stats or perks info to be expanded on in that same document.

gothfemme commented:
> I wasn't too sure how you wanted the page to look so I started with a little info on navigating the roll editor. I added anchors to some of the other headers as I imagine this page can and will get quite long if you wanted say the stats or perks info to be expanded on in that same document.

You actually don't need to add the anchor tags! They'll be auto-generated from the markdown headers :)

JayAndromeda commented:
updated to remove anchors and changed the markdown links to use the ones generated by the markdown header... also removed the word 'just' since it was a linting error.

gothfemme reviewed (approved):
Looks great!! Thank you for writing this up
